org.eclipse.wildwebdeveloper/language-servers/.gitignore

```diff
@@ -1,4 +1,5 @@
 /node_modules/
+!/node_modules/.bin
```

Comment:
I don't think we should put binaries on SCM herel it's a bad practice and is adding 1 extra set of artifacts to think about maintaining. Those should be provisioned at build time instead.

Reply:
> I don't think we should put binaries on SCM herel it's a bad practice and is adding 1 extra set of artifacts to think about maintaining. Those should be provisioned at build time instead.

Feel free to change, but I'm unable to see a simpler alternative (and don't know enough about npm to make it behave "a bit like windows" (i.e., create scripts not links) while running on linux).
I also wouldn't consider scripts as binaries :)

Reply:
Right, I used the wrong terms. It's more that I'm reluctant to check-in 3rd party code in general.